client/my-sites/purchases/index.js

```diff
@@ -110,10 +110,10 @@ export default ( router ) => {
 	);
 
 	page(
-		'/purchases/add-credit-card/:site',
+		'/purchases/add-payment-method/:site',
```

Comment:
At one point I had both and then I decided to change it for everyone. In retrospect, it's probably better to keep all the new updates behind the flag and otherwise keeps things as is for everyone else.